rules/python_rules.build_defs

```diff
@@ -531,12 +562,23 @@ def python_wheel(name:str, version:str, hashes:list=None, package_name:str=None,
     if post_install_commands:
         cmd += post_install_commands
 
-    cmd += ['$TOOL z -d --prefix $PKG -i ' + ' -i '.join(outs or [name])]
+    if use_package_root:
+        jarcat_command = '$TOOL z -d -i ' + ' -i '.join(outs or [name])
+    else:
+        jarcat_command = '$TOOL z -d --prefix $PKG -i ' + ' -i '.join(outs or [name])
+
+    cmd += [jarcat_command]
```

Comment:
"I'm not sure why we need this here? Third party code can already be imported via the root path."

Because implicitly the content of the wheel gets added into the zip, being prefixed by the package name where is defined. So in the zip you won't have the folder `flask`, but rather `third_party/python/flask` (note the `--prefix $PKG`). 

The import via the root path is only possible if the `modulesdir` is configured to that package name in the please config file. Since the modulesdir is defined, the custom module loader in the pex also searches that location, so the imports will look like they're done from the rootpath. In the pex file, they will still reside in the folder matching the package name, not in the root of the archive.

It seems more reasonable to have a single switch that configures everything to be in the root, and use regular imports, since it's also closer to how python operates - namely just unpacking all the packages into its lib folder, and read from there.